server/src/encrypt.py

```diff
@@ -0,0 +1,14 @@
+from cryptography.fernet import Fernet
+
+
+def encrypt(key: str, msg: str) -> str:
+    fernet = Fernet(key)
```

Comment:
Consider to write something like this
```suggestion
    fernet = Fernet(SECRET)
```

If there are always used SECRET from settings - there are no need to pass it like function argument